### PR TITLE
fix(build): fixes build process for split project

### PR DESCRIPTION
**Pull request by fzipi**
Signed-off-by: Felipe Zipitria <felipe.zipitria@owasp.org>

Fixes build after splitting repos.

Next step might be moving the build to github actions.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -127,3 +127,12 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Upstream git repo
+upstream/
+libinjection/libinjection.h
```
Comment (jpereira) [inline .gitignore:133]: Maybe `*.pyc` also?

File: Makefile [hunk 2]
```diff
@@ -21,29 +21,31 @@ test: test-unit
 speed:
 	./speedtest.py
 
+upstream: 
+	[ -d $@ ] || git clone --depth=1 https://github.com/libinjection/libinjection.git upstream
 
-words.py: Makefile json2python.py ../src/sqlparse_data.json
-	./json2python.py < ../src/sqlparse_data.json > words.py
+libinjection/libinjection.h libinjection/libinjection_sqli.h: upstream
+	cp upstream/src/libinjection*.h upstream/src/libinjection*.c libinjection/
```
Comment (jpereira) [inline Makefile:28]: Maybe `cp -f`

**Review by jpereira — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (fzipi) [reply .gitignore:133]: For sure. That is already in the `.gitignore` file [here](https://github.com/libinjection/python3-libinjection/blob/84b39d598056a01a0860c659dcfc33beeb79fff5/.gitignore#L3). We are only seeing the addition/changes here. :)

File: Makefile (hunk 2 above)
Comment (fzipi) [reply Makefile]: Ammended change.

**Review by jpereira — APPROVED** [review]

**Comment by zimmerle** [comment]
@fzipi @jpereira I am sorry for the delay.

**Comment by zimmerle** [comment]
Thank you @fzipi

**Pull request merged**